app/assets/javascripts/browsered/web-payments/apple-pay.js

```diff
@@ -30,7 +31,6 @@ module.exports = () => {
       .then(response => {
         session.completeMerchantValidation(response)
       }).catch(err => {
-        session.abort()
```

Comment:
why does this line need to be removed?

Reply:
Because it raised an error, when I run locally (where I wont validate as a merchant) and this got called in caused an error so the JS and so the error message below doesn’t show. Basically it’s too early to call abort here, becuase a valid Apple Pay session hasn’t begun